flink-end-to-end-tests/test-scripts/test_quickstarts.sh

```diff
@@ -68,6 +68,13 @@ sed -i -e ''$(($position + 1))'i\
 
 sed -i -e "s/org.apache.flink.quickstart.StreamingJob/org.apache.flink.quickstart.$TEST_CLASS_NAME/" pom.xml
 
+if [[ $PROFILE == *"scala-2.12"* ]]; then
+  echo "Changing scala version"
+  sed -i -e "s/scala.binary.version>2.11<\/scala.binary/scala.binary.version>2.12<\/scala.binary/" pom.xml
+  # for flink-quickstart-scala, also change scala version
+  sed -i -e "s/scala.version>2.11.12<\/scala.ver/scala.version>2.12.7<\/scala.ver/" pom.xml
+fi
+
```

Comment:
maybe add bashy equivalent of
```
else if profile == 2.11 do nothing
else throw new UnsupportedOperationException()
```
?

Reply:
That would be way too clean for our e2e tests :) 
In case of an unsupported scala version, the e2e test would fail through maven in the scala compile stage.

Addressed in https://github.com/rmetzger/flink/commit/eda372b26dc5ea82b03f0ada8d16bc85afc61123

Reply:
Isn't it as it is a ticking time bomb once we add support for scala 2.13?

Reply:
Fair point. I changed the code.